### PR TITLE
bump emitter to v0.32

### PR DESCRIPTION
**Pull request by MaryGao**

### CODE REVIEW

File: packages/autorest.typescript/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+- [Feature] Support Modular readme.md generation. Please refer to [#2655](https://github.com/Azure/autorest.typescript/pull/2655).
 - [Feature] Engineering system related improvements.
-    - Bump typescript version to ~5.5.3, please refer to [#2646](https://github.com/Azure/autorest.typescript/pull/2646); 
-    - vitest timeout config, please refer to [#2632](https://github.com/Azure/autorest.typescript/pull/2632); 
-    - recorded client for esm package improve, please refer to [#2632](https://github.com/Azure/autorest.typescript/pull/2632);
-    - skip sample validation if generateSample is false, please refer to [#2650](https://github.com/Azure/autorest.typescript/pull/2650); 
-    - remove unused configuration files, please refer to [#2655](https://github.com/Azure/autorest.typescript/pull/2655); 
-    - add and refine document for generated code, please refer to [#2661](https://github.com/Azure/autorest.typescript/pull/2661) and [#2621](https://github.com/Azure/autorest.typescript/pull/2621).
+  - Bump typescript version to ~5.5.3, please refer to [#2646](https://github.com/Azure/autorest.typescript/pull/2646);
```
Comment (MaryGao) [inline packages/autorest.typescript/CHANGELOG.md:21]: it seems the format is correct: https://github.com/Azure/autorest.typescript/blob/1d1acc18fb4d23174f1dfea41b0dcc43420847ac/packages/autorest.typescript/CHANGELOG.md
image

**Comment by qiaozha** [comment]
Can you check if we can wait for this PR to be merged https://github.com/Azure/autorest.typescript/pull/2751

File: packages/autorest.typescript/CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,30 @@
+## 6.0.26 (2024-08-15)
+
+- [Feature] Remove internal RLC layer from modular. Please refer to [#2728](https://github.com/Azure/autorest.typescript/pull/2728).
+- [Feature] Upgrade compiler to v0.59. Please refer to [#2741](https://github.com/Azure/autorest.typescript/pull/2741).
```
Comment (lirenhe) [inline packages/autorest.typescript/CHANGELOG.md:4]: TypeSpec compiler to 0.59

**Review by lirenhe — APPROVED** [review]
One wording issue,  other part looks good.

File: packages/rlc-common/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
-## 0.32.0 (2024-08-15)
+## 6.0.26 (2024-08-15)
```
Comment (qiaozha) [inline packages/rlc-common/CHANGELOG.md:1]: 6.0.26 for rlc common ?

File: packages/typespec-ts/CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-## 0.32.0 (2024-08-15)
+## 6.0.26 (2024-08-15)
```
Comment (qiaozha) [inline packages/typespec-ts/CHANGELOG.md:1]: same here.

**Review by qiaozha — CHANGES_REQUESTED** [review]
please don't copy the autorest.typescript change log into rlc-common or typespec-ts folder

**Review by qiaozha — APPROVED** [review]

**Pull request merged**